src/app/models/project.ts

```diff
@@ -52,6 +52,9 @@ export class Project {
   projectLead: any;
   projectDirector: any;
 
+  // Options
+  activitiesAndUpdatesEnabled: Boolean;
```

Comment:
I know there are examples of `Boolean` as a type in this app but it's actually incorrect. It should still work but the correct type would be the lowercase equivalent. `Boolean` refers to the object, whereas `boolean` is the primitive. This goes for other primitives too(`string`, `number`, etc.)